lib/modules/dosomething/dosomething_reportback/includes/Reportback.php

```diff
@@ -92,6 +92,14 @@ public static function get($ids) {
 
     $results = dosomething_reportback_get_reportbacks_query(['rbid' => $ids]);
 
+    if (!user_access('administer modules') && !$results) {
+      $results = dosomething_reportback_get_reportbacks_query(['rbid' => $ids], TRUE);
```

Comment:
Oh wait. I was misreading it. Still can't setting it to true happen in the `dosomething_reportback_get_reportbacks_query()` function?